src/main/java/uk/gov/companieshouse/company_appointments/model/data/CompanyAppointmentData.java

```diff
@@ -13,9 +13,12 @@ public class CompanyAppointmentData {
 
 	private OfficerData data;
 
-	public CompanyAppointmentData(String id, OfficerData data) {
+	private String company_status;
```

Comment:
This should be camel case, you can write it like this: 
```suggestion
        @Field("company_status")
	private String companyStatus;
```